controllers/workloads/console/integration/integration_test.go

```diff
@@ -241,15 +241,44 @@ var _ = Describe("Console", func() {
 				"job's pod restartPolicy should always be set to 'Never'",
 			)
 			Expect(
-				job.Spec.Template.Spec.Containers[0].Stdin).To(BeTrue(),
-				"job's first container should have stdin true",
+				job.Spec.Template.Spec.Containers[0].Stdin).To(BeFalse(),
+				"job's first container should have stdin false",
 			)
 			Expect(
-				job.Spec.Template.Spec.Containers[0].TTY).To(BeTrue(),
-				"job's first container should have tty true",
+				job.Spec.Template.Spec.Containers[0].TTY).To(BeFalse(),
+				"job's first container should have tty false",
 			)
 		})
 
+		Context("with Interactive = true", func() {
```

Comment:
Mulling over whether this should be true by default or not. Arguably it doesn't matter too much as the consumer of the library can make the decision to opt-in to TTYs (we can certainly do that in our internal tooling that needs them enabled).

Trying to figure out which way round is less of a surprise as a user of the library.

Reply:
I think that's the caller's decision - anu consoles create defaults it to true in our test branch, FWIW. Utopia should do the same. If we do more automation in this style, however, we could potentially have more places than Dispatcher where `false` made sense.

Reply:
See https://github.com/gocardless/theatre/pull/211#discussion_r483461870